src/bp/ui-studio/src/web/components/Confusion/index.jsx

```diff
@@ -0,0 +1,114 @@
+import { FaThLarge } from 'react-icons/fa'
```

Comment:
Typescript is now available in the studio, try to create new files as tsx instead of jsx